core/dbt/common/events/event_manager_client.py

```diff
@@ -0,0 +1,29 @@
+# Since dbt-rpc does not do its own log setup, and since some events can
+# currently fire before logs can be configured by setup_event_logger(), we
+# create a default configuration with default settings and no file output.
+from dbt.common.events.event_manager import IEventManager, EventManager
+
+_EVENT_MANAGER: IEventManager = EventManager()
+
+
+def get_event_manager() -> IEventManager:
```

Comment:
this functionality was moved from `functions.py` to make it clearer how the event manager is setup and managed